tests/test_crystal_ball.py

```diff
@@ -1,15 +1,39 @@
 """Module for testing the htcrystalball module."""
+
 import argparse
 
-from os import path
-from pytest import fixture, raises as praises
+from pytest import raises as praises
 
 from htcrystalball import examine, collect, utils
 
 
-@fixture
-def root_dir() -> str:
-    return path.dirname(path.abspath(__file__))
+COLL_QUERY = [
```

Comment:
I strongly suggest using mocking rather than hardcoding output. The purpose of mocking is not just capturing API output. It's also to make sure that the tests and production code use the same code paths. Otherwise the tests are missing essential parts of the project code base.

Reply:
Yes, I get it. But the only thing that I am now missing is coll = HTCondor.Collector() and coll.query(some_params). So what essential part of the code base are the tests now missing?

Reply:
I will see if I can mock `coll = HTCondor.Collector()` and `coll.query(some_params)`